frontend/pages/hosts/details/OSSettingsModal/OSSettingsTable/OSSettingStatusCell/helpers.ts

```diff
@@ -22,28 +22,35 @@ type OperationTypeOption = Record<
 
 type ProfileDisplayConfig = Record<ProfileOperationType, OperationTypeOption>;
 
+const MAC_PROFILE_VERIFIED_DISPLAY_CONFIG: ProfileDisplayOption = {
+  statusText: "Verified",
+  iconName: "success",
+  tooltip: (innerProps) =>
+    innerProps.isDiskEncryptionProfile
+      ? "The host turned disk encryption on and sent the key to Fleet. " +
+        "Fleet verified with osquery."
+      : "The host applied the setting. Fleet verified with osquery. " +
+        "Declaration profiles are verified with DDM.",
+};
+
+const MAC_PROFILE_VERIFYING_DISPLAY_CONFIG: ProfileDisplayOption = {
+  statusText: "Verifying",
+  iconName: "success-outline",
+  tooltip: (innerProps) =>
+    innerProps.isDiskEncryptionProfile
+      ? "The host acknowledged the MDM command to turn on disk encryption. " +
+        "Fleet is verifying with osquery and retrieving the disk encryption key. " +
+        "This may take up to one hour."
+      : "The host acknowledged the MDM command to apply the setting. Fleet is " +
+        "verifying with osquery.",
+};
```

Comment:
Can these be designated `as const`?

Reply:
yep i've added. but im not sure what this is giving us in this case. I feel like I'm missing something here